webpack.config.extensions.js

```diff
@@ -72,76 +60,14 @@ function getWebpackConfig() {
 		...blockScripts( 'editor', path.join( __dirname, 'extensions' ), allPresetBlocks ),
 	];
 
-	const webpackConfig = {
-		bail: ! isDevelopment,
-		context: __dirname,
+	return getBaseWebpackConfig( null, {
 		entry: {
```

Comment:
```suggestion
		entry: {
		'index.json': 'extensions/setup/index.json',
```

Reply:
This works _almost_ -- webpack appends `.js` to `index.json` :sweat_smile:

Reply:
(And it does weird webpack things to it)